src/main/java/Entity/GameBoard.java

```diff
@@ -0,0 +1,102 @@
+package Entity;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class GameBoard {
+    public List<Integer> players;
+    public Integer[] board = new Integer[40];
+    //private int goPosition = 0;
+    //private int jailPosition = 30;
+    //private int[] chanceCard = new int[]{7, 22, 36};
+    //  private int[] oppurtunityCard = new int[]{2, 17, 33};
+    static int turn = 0;
+
+    public GameBoard(){
+
+    }
+
+
+    // monopoly overall class that will keep game organized
+
+    public GameBoard(Player inputPlayers, Properties[] lstProperties){
+        this.players = inputPlayers;
+        this.board = lstProperties;
+
+    }
+
+    public void addPlayer(PLayer player1){
+        players.add(player2);
```

Comment:
players.add(player1); ?

Reply:
Yes, it should be player1, thanks for catching that.  There is also a typo in the method that I will fix